docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -385,6 +431,85 @@ Finally, the second-level cache can be disabled globally by setting `hibernate.c
 
 When second-level cache is disabled, all cache annotations are ignored and all queries are run ignoring caches; this is generally useful only to diagnose issues.
 
+== Limitations and other things you should know
+
+Quarkus does not modify the libraries it uses; this rule applies to Hibernate ORM as well: when using
+this extension you will mostly have the same experience as using the original library.
+
+But while they share the same code, Quarkus does configure some components automatically and injects custom implementations
+for some extension points; this should be transparent and useful but if you're an expert of Hibernate you might want to
+know what is being done.
+
+=== Automatic build time enhancement
+
+Hibernate ORM can use build time enhanced entities; normally this is not mandatory but it's useful and will have your
+applications perform better.
+
+Typically you would need to adapt your build scripts to include the Hibernate Enhancement plugins; in Quarkus this is
+not necessary as the enhancement step is integrated in the build and analysis of the Quarkus application.
+
+
+=== Automatic integration
+
+Transaction Manager integration::
+You don't need to set this up, Quarkus automatically injects the reference to the Narayana Transaction Manager.
+The dependency is included automatically as a transitive dependency of the Hibernate ORM extension.
+All configuration is optional; for more details see link:transaction.html[Using Transactions in Quarkus].
+
+Connection pool::
+Don't need to choose one either. Quarkus automatically includes the Agroal connection pool;
+just configure your datasource as in the above examples and it will setup Hibernate ORM to use Agroal.
+More details about this connection pool can be found in link:datasource-guide.html[Quarkus - Datasources].
```

Comment:
This URL is outdated, it should point to: https://quarkus.io/guides/datasource

There is a redirect so it's no biggie but better get this straight.

Reply:
ah thanks!  well, without the `.html` too :)

Reply:
Yeah, I will make a pass on all the guides to remove the `.html`. That's why I didn't mention it.